packages/react-native/plugins/metro-resolver.ts

```diff
@@ -136,17 +134,14 @@ function tsconfigPathsResolver(
         )
       );
     }
-    throw new Error(`Cannot resolve ${chalk.bold(moduleName)}`);
   }
 }
 
 let matcher: MatchPath;
 let absoluteBaseUrl: string;
 let paths: Record<string, string[]>;
 
-function getMatcher() {
-  const DEBUG = process.env.NX_REACT_NATIVE_DEBUG === 'true';
-
+function getMatcher(DEBUG: boolean) {
```

Comment:
Nitpick: Arguments and variables should be mixed case (`debug`).

Reply:
done